app/build.gradle

```diff
@@ -208,6 +208,7 @@ dependencies {
       'androidx.test.ext:junit:1.1.1',
       'com.github.bumptech.glide:mocks:4.11.0',
       'com.google.truth:truth:1.1.3',
+      'androidx.work:work-testing:2.4.0',
```

Comment:
Wasn't this introduced in your other PR? I suggest syncing to the latest develop so that unrelated changes don't need to be in this branch.

Reply:
Yeah @BenHenning updated and synced branch to pick up latest develop.